### DISCUSSION TITLE
Archer T4U Plus

### DISCUSSION THREAD

**Discussion by ASC-David** [chris1111/Wireless-USB-Big-Sur-Adapter#194] [Q&A]
I'm currently running Monterey 12.2.1 on a MacPro5,1 that does not have internal WiFi or bluetooth. My Archer T3U Plus works perfectly with this driver and procedure - thanks!. Question: Will the Archer T4U Plus work? The list says that ArcherT4U V1, V2, V3 work, but the Plus also contains bluetooth. If confirmed to not work, I don't want to buy it. If unconfirmed either way, I'm willing to give it a shot and report back - what are the odds that it will work, or not?

Thanks in advance, DT

**Comment by chris1111** [comment]
bluetooth from USB Wifi adapter never work on macOS, only in windows. If you whant bluethhot you have to buy a compatible dongle bluetooth

**Comment by ASC-David** [comment]
The specs for it show it’s supposedly good for Mac OS 10.9 - 10.14, which would be Mojave, right? I haven’t been able to find anything that indicates the bluetooth won’t work on a Mac. Under Mojave, I was using separate usb wifi and bluetooth, but neither work since Catalina. As I mentioned, my T3U Plus works fine via Open Core. I’d like to eventually get them running on one device.

Any other comments or suggestions welcome.
_________________________________ 
David Truran
A Sound Collective
Digital Precision, Analog Sensibility!
https://www.facebook.com/ASCTruran/

**Reply by chris1111** [reply]
>  I haven’t been able to find anything that indicates the bluetooth won’t work on a Mac. 


But you have not seen and will not see that bluethoot works in macOS
I have never seen anyone tell me that the bluetooth of their wifi dongle works in macOS

**Comment by chris1111** [comment]
Edimax indicate that the bluethoot not working in macOS, I think its the only compagny who says it very clearly
![Screen Shot 2022-03-06 at 9 30 34 PM](https://user-images.githubusercontent.com/6248794/156957735-98e842d6-e0dd-4be2-8ea5-8be13476f4b3.png)
[see pdf file](https://www.edimax.com/edimax/mw/cufiles/files/download/QIG/EW-7611UCB/EW-7611UCB_QIG_EN.pdf)

**Comment by ASC-David** [comment]
Thanks!
_________________________________ 
David Truran
A Sound Collective
Digital Precision, Analog Sensibility!
https://www.facebook.com/ASCTruran/
